scripts/unix_share_build.sh

```diff
@@ -74,6 +74,8 @@ function setup_var() {
         CUBIT_PATH="/opt/Coreform-Cubit-2021.3"
     elif [ "$1" == "2021.4" ] ; then
         CUBIT_PATH="/opt/Coreform-Cubit-2021.4"
+    elif [ "$1" == "2022.4" ] ; then
+        CUBIT_PATH="/opt/Coreform-Cubit-2022.4"
```

Comment:
this is not used, but I thought it might be better for consistency

Reply:
Why isn't it used? At least it avoids the exit condition, right?

Reply:
it is only used if we run this using the build scripts and not the CI.